syntax = "proto2";

package opentxs.proto;
option java_package = "org.opentransactions.proto";
option java_outer_classname = "OTOutBailment";
option optimize_for = LITE_RUNTIME;

message OutBailment {
    optional uint32 version = 1;
    optional string unitID = 2;
    optional string serverID = 3;
    optional uint64 amount = 4;
    optional string instructions = 5;
    repeated string payment = 6;
}
